kata-1/variables.tf

```diff
@@ -0,0 +1,44 @@
+variable "namespace" {
+  description = "Project namespace for unique resource naming"
+  type        = string
+}
+
+
+variable "region" {
+  description = "AWS Region"
+  default     = "us-east-1"
+  type        = string
+}
+
+variable "az" {
+  description = "AWS Availability Zone"
+  default     = "us-east-1a"
+  type        = string
+}
+
+variable "ami" {
+  description = "Amazon Machine Image"
+  type        = string
+  default     = "ami-0947d2ba12ee1ff75" # Amazon Linux 2 AMI (HVM), SSD Volume Type
+}
+
+variable "vpc_cidr" {
+  description = "The CIDR block for VPC"

```

Comment:
consider giving a default value as the requirements are specific to a certain cidr range

Reply:
Will do

Reply:
Added default value for cidr range on [this ](https://github.com/adriaanbd/terraform_aws_dojo/pull/1/commits/c3047f1b7ce697205eb1d341b811fe8bfc34b4d0) commit